keps/sig-storage/3141-prevent-volume-mode-conversion/README.md

```diff
@@ -468,13 +512,17 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+N/A.
```

Comment:
This would be a little clearer if it indicated that gating the PVC creation does not impact the ability of the pod to use the PVC already created.

Can one of these snapshots be used from inside of a stateful set?  if so, this would impact scaling and updates, right?